protostar/cheatable_starknet/controllers/contracts.py

```diff
@@ -119,10 +133,48 @@ async def _transform_calldata_to_cairo_data(
                 ) from dt_exc
         return calldata or []
 
+    async def declare_sierra_contract(
```

Comment:
Why do we need this one in `PreparedContract` 🤔

Reply:
It's ContractsController. Diff is just weird.